.gitignore

```diff
@@ -0,0 +1,141 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This is a standard file to make sure you only commit the things you want.  In particular, it makes sure to ignore the build artifacts when you build and install the python package into a conda environment.

Reply:
This is very useful as when I pushed the initial version to github, I had to manually remove files.